### ISSUE TITLE
docker container fails with `importlib.metadata.PackageNotFoundError: goosepaper` 

### ISSUE THREAD

**Issue by v0d** [j6k4m8/goosepaper#20]
I've tried to run goosepaper via docker, but the docker container seems to lack (or python within the container cant locate) the goosepaper package:

```
% docker run -it --rm -v $(pwd):/goosepaper j6k4m8/goosepaper goosepaper -c example-config.json
Traceback (most recent call last):
  File "/usr/local/bin/goosepaper", line 33, in <module>
    sys.exit(load_entry_point('goosepaper', 'console_scripts', 'goosepaper')())
  File "/usr/local/bin/goosepaper", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
  File "/usr/local/lib/python3.8/importlib/metadata.py", line 504, in distribution
    return Distribution.from_name(distribution_name)
  File "/usr/local/lib/python3.8/importlib/metadata.py", line 177, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: goosepaper
```

**Comment by j6k4m8** [comment]
Thank you for reporting, @v0d! Turns out this was just a silly documentation error on my part. (By way of short explanation, the `-v` flag I wrote in the docs overwrites the goosepaper codebase inside the container..... whoops.)

The fix:

```shell
docker run -it --rm -v $(pwd):/goosepaper/mount j6k4m8/goosepaper goosepaper -c mount/example-config.json -o mount/Goosepaper.pdf
```

In other words, mount the current directory to `$(pwd):`**`/goosepaper/mount`** and point to configs inside that directory. So if you're in a folder with a file called `goosepaper-config.json`, your command now looks like:

```shell
docker run -it --rm -v $(pwd):/goosepaper/mount j6k4m8/goosepaper goosepaper -c mount/goosepaper-config.json -o mount/Goosepaper.pdf
```

Sorry for the confusion! Let me know if that fixes it for you and I'll update the docs too :)
